Sources/AppCoinsSDK/UI/Login/MagicLinkCodeView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  MagicLinkCodeView.swift
+//
+//
+//  Created by Graciano Caldeira on 28/11/2024.
+//
+
+import SwiftUI
+
+struct MagicLinkCodeView: View {
+    
+    @ObservedObject internal var viewModel: BottomSheetViewModel
+    
+    var body: some View {
+        if #available(iOS 17, *) {
+            ZStack(alignment: .top) {
+                ScrollViewReader { scrollViewProxy in
+                    ScrollView(.vertical, showsIndicators: false) {
+                        VStack(spacing: 0) {
+                            
+                            VStack {}.frame(height: 56)
```

Comment:
Do not leave this spacing between 'VStack' and the '{}'

Reply:
Done